react/ui-components/src/atoms/SelectionTag.js

```
@@ -14,12 +14,47 @@ const SelectionTag = ({
   allowMultipleSelection = true,
   selected,
   withContainer,
-  populators={}
+  populators = {},
 }) => {
   const { t: i18nT } = useTranslation();
   const t = populators?.t || i18nT;
   const [selectedOptions, setSelectedOptions] = useState(selected || []);
 
+  const { isLoading, data } = window?.Digit?.Hooks.useCustomMDMS(
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)